driver/pace/driver/run.py

```diff
@@ -234,9 +235,7 @@ def _get_grid_data_damping_coeff_and_driver_grid(
             )
             grid_data = grid.grid_data
         else:
-            grid = fv3core._config.make_grid_with_data_from_namelist(
-                self._namelist, communicator, backend
-            )
+            grid = Grid.with_data_from_namelist(self._namelist, communicator, backend)
```

Comment:
Actually, I only did this so that the returns are the same as using serialized_grid, but it's actually never used and can just be `grid = None`.

Reply:
Okay a better refactor should be to return grid_data instead of grid in `_get_serialized_grid_damping_coeff_and_driver_grid`. You can leave them as it is, and I can open another PR to clean this up.